addon/components/au-accordion.hbs

```diff
@@ -4,24 +4,24 @@
       <div>
         <AuButton @skin="tertiary" aria-hidden="true" focusable="false" aria-expanded="{{if this.openAccordion "true" "false"}}">
           {{#if @hideText}}
-            <span class="au-u-hidden-visually">{{@accordionButtonLabel}}</span>
+            <span class="au-u-hidden-visually">{{@label}}</span>
           {{else}}
-            {{@accordionButtonLabel}}
+            {{@label}}
           {{/if}}
         </AuButton>
         <p>
-          {{@accordionSubTitle}}
+          {{@subTitle}}
         </p>
       </div>
     </AuToolbarGroup>
     <AuToolbarGroup>
       {{#if this.accordionOpen}}
-        <AuIcon @icon="{{@accordionIconOpen}}" @alignment="left" @size="large" @ariaHidden="true" />
+        <AuIcon @icon="{{if @iconOpen @iconOpen 'nav-up'}}" @alignment="left" @size="large" @ariaHidden="true" />
```

Comment:
I would do this in a getter in the backing class. That way you can also take the open state into account and simplify this whole if / else structure.

> Note: setting icon to false doesnt remove it from html but just doesnt assign an icon so it displays nothing

I think passing in `{{false}}` would return `nav-up` instead of an empty icon? I'm guessing you meant the string "false" and that would be true for any invalid icon name I suppose. 

Good point though, should it be possible to not show the icons? In that case an extra "showIcons" flag (which defaults to true) might be better? (and not show the icon component at all)